src/system/RoutesModule/Resources/config/customServices.yml

```diff
@@ -20,17 +23,17 @@ services:
     zikula_routes_module.routing_loader:
         class: Zikula\RoutesModule\Routing\RouteLoader
         arguments:
-            - "@doctrine.orm.entity_manager"
-            - "@service_container"
+            - "@doctrine.orm.default_entity_manager"
+            - "@translator"
```

Comment:
No, `translator` is correct here, since the route loader uses the Symfony translator, not the special one from Zikula.